coderabbit/e2ecopilotutil/copilot/metricsapi_utils.py

```
@@ -0,0 +1,91 @@
+        f"Waiting up to {timeout} seconds for {gwname} to appear in MetricsAPI output..."
+    )
+    out = None
+    while time.time() < stopTime:
+        _, out = ctrl.run(cmd)
+        if gwname in out:
+            break
+        else:
+            logger.debug(out)
+            time.sleep(10)
+    else:
+        assert False, f"MetricsAPI failed to report stats for {gwname}!"
```

Ruff (0.8.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)